wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Scrolling.swift

```diff
@@ -52,11 +52,24 @@ extension ConversationContentViewController {
         updateTableViewHeaderView()
     }
 
-    func scrollToBottom() {
-        guard !isScrolledToBottom else { return }
+    /// Scrolls the tableView to the bottom-most row.
+    ///
+    /// This method checks if the tableView is not already scrolled to the bottom.
+    func scrollToBottomIfNeeded() {
+        let maxMessageCountForAnimation = 20
+
+        guard !isScrolledToBottom else {
+            return
+        }
 
         dataSource.loadMessages()
-        tableView.scroll(toIndex: 0)
+
+        let lastRowIndexPath = IndexPath(row: 0, section: 0)
+        let shouldAnimate = !UIAccessibility.isReduceMotionEnabled && dataSource.messages.count <= maxMessageCountForAnimation
+
+        let scrollAnimation: UITableView.ScrollPosition = shouldAnimate ? .bottom : .top
```

Comment:
Why does the scroll position change depending whether we should animate?

Reply:
It was a leftover as I was trying different solutions. It's not needed indeed. Good catch @johnxnguyen